packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -1937,6 +1937,10 @@ export class StripeHelper extends StripeHelperBase {
             plan: newPlanId,
           },
         ],
+        proration_behavior: this.config.subscriptions.stripeInvoiceImmediately
+          .enabled
+          ? 'always_invoice'
+          : 'create_prorations',
```

Comment:
This didn't work for me when setting the value directly. `proration_behavior` didn't show up in the VSCode debugger for me at all. Pulling it out did, like so:

```
const prorationBehavior = this.config.subscriptions.stripeInvoiceImmediately
  .enabled
  ? 'always_invoice'
  : 'create_prorations';

const updatedSubscription = await this.updateSubscriptionAndBackfill(
  subscription,
  {
    cancel_at_period_end: true,
    items: [
      {
        id: subscription.items.data[0].id,
        plan: newPlanId,
      },
    ],
    proration_behavior: prorationBehavior,
    metadata: updatedMetadata,
  }
);
```
If you happen to know why, let me know, otherwise, to be safe, I'd say we should do it like above. It's temporary, but also consistent with how `metadata` is being set.

Reply:
done 👍